journal/original/20-chapter-17.adoc

```diff
@@ -1191,6 +1227,9 @@ and it was not known whether they were clear and orderly, or not.
 And it opened in me to recommend the following method
 unto Friends for preventing such inconveniences:--
 
+[.embedded-content-document.letter]
```

Comment:
Because he's addressing Friends in general on the subject of marriage, let's go `epistle`.